dcmgr/lib/dcmgr/models/instance.rb

```diff
@@ -37,6 +37,7 @@ class Instance < AccountResource
     subset(:runnings, {:state => STATE_RUNNING})
     subset(:running_or_initializing, {:state => [STATE_RUNNING, STATE_INITIALIZING]})
     subset(:stops, {:state => STATE_STOPPED})
+    subset(:expired) { (password_will_be_deleted_at > 0) && (password_will_be_deleted_at <= Time.now) }
 
```

Comment:
This line seems to work when I test it, but I'm not 100% sure if this is OK, because password_will_be_deleted_at can be nil.  Can some expert on Sequel comment on this?